### PR TITLE
updating material alpha

### PR DESCRIPTION
**Pull request by mrieggeramzn**
Updating decal materials with better settings so that they are visible in the Material Editor

checked ASV to make sure decal asv test still runs ok

ATOM-15909

Signed-off-by: mrieggeramzn <mriegger@amazon.com>

### CODE REVIEW

File: Materials/decal/am_mud_decal.material [hunk 1]
```diff
@@ -17,11 +17,9 @@
...
         },
         "opacity": {
-            "alphaSource": "Split",
-            "doubleSided": true,
-            "factor": 0.6899999976158142,
-            "mode": "Cutout",
-            "textureMap": "Materials/decal/am_mud_decal.tif"
+            "factor": 1.0,
+            "mode": "Blended",
+            "textureMap": "am_mud_decal.tif"
```
Comment (Adi-Amazon) [inline Materials/decal/am_mud_decal.material:22]: was the texture move to the root? why?

File: Materials/decal/scorch_01_decal.material [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
     "description": "",
-    "materialType": "Materials\\Types\\StandardPBR.materialtype",
     "parentMaterial": "",
-    "propertyLayoutVersion": 3,
+    "materialType": "Materials/Types/StandardPBR.materialtype",
+    "materialTypeVersion": 4,
     "properties": {
         "baseColor": {
-            "textureMap": "Materials/decal/scorch_01_decal.tif"
+            "textureMap": "scorch_01_decal.tif"
```
Comment (Adi-Amazon) [inline Materials/decal/scorch_01_decal.material:8]: moved to the root directory?

**Review by Adi-Amazon — CHANGES_REQUESTED** [review]
Not a blocker, but per my question - were the two textures moved to the root directory rather than Materials/decal?
This is not in this CR if so

File: Materials/decal/am_mud_decal.material (hunk 1 above)
Comment (mrieggeramzn) [reply Materials/decal/am_mud_decal.material:22]: So I really just loaded, modified the alpha settings, and resaved. The material editor seemed to have made those changes. Also, I could reload that material afterwards (stopping and starting the materialeditor) and it seems to have worked ok. 

I just assumed the MaterialEditor knew more about paths than I did/

**Review by Adi-Amazon — APPROVED** [review]

File: Materials/decal/scorch_01_decal.material (hunk 2 above)
Comment (Adi-Amazon) [reply Materials/decal/scorch_01_decal.material:8]: Assuming the editor work method started to look at local directory and remove the relative directory

**Review by wintermute-motherbrain — APPROVED** [review]

**Pull request merged**